connect_ext_ppr/models/utils.py

```diff
@@ -4,35 +4,35 @@
 
 def add_deployments(installation, listings, config, logger):
```

Comment:
I think we should move this to a service file. It's more like a service than a utility.